client/bindings/wasm/src/lib.rs

```diff
@@ -16,7 +16,7 @@ pub fn start() -> Result<(), JsValue> {
 
 /// The Wasm bindings do not support internal logging yet.
 ///
-/// Calling this is a no-op, only included for compatibility with the Neon Node.js bindings TypeScript definitions.
+/// Calling this is a no-op, only included for compatibility with the Node.js bindings TypeScript definitions.
```

Comment:
Well it's there for compatibility with the neon nodejs bindings :) but no big deal